src/components/graph-controls.js

```diff
@@ -27,8 +27,8 @@ import Radium from 'radium';
 import PropTypes from 'prop-types';
 
 // Only old-style imports for react-icons seem to work with gulp
-const FaExpand = require('react-icons/lib/fa/expand');
-const FaInfoCircle = require('react-icons/lib/fa/info-circle');
+import {FaExpand} from 'react-icons/fa';
```

Comment:
I get "can't resolve module `react-icons/fa` in VSCode, do you know if there needs additional changes to get VSCode to accept the new `import from react-icons/fa`?

Reply:
I'm not sure. It's hard for me to replicate this because I can't install the deps for the project because internal `d3` is being used

Reply:
I'm not sure what you mean by "internal d3", whether that's your own d3 code (internal to you), or whether you mean it's added as a dependency now rather than a peerDependency. I'll need a little more information before I can help there. Perhaps you can delete your `node_modules` directory in the project and perform a new `npm install`.

Reply:
https://github.com/uber/react-digraph/blob/master/yarn.lock#L1901-L1903

Reply:
`curl https://unpm.uberinternal.com/d3/-/d3-4.13.0.tgz` returns 401

Reply:
Ahh, internal to Uber using unpm. You might want to give it a try with `npm` instead of `yarn`. I believe the Yarn file was added just recently and we didn't add the yarn.lock file to the gitignore file. I've done this now and published 4.1.4 with the new gitignore file.